packages/ap-codegen/alliance_platform/codegen/printer.py

```diff
@@ -107,6 +107,29 @@ def wrapper(self: "TypescriptPrinter", node: NodeLike, *args, **kwargs):
 )
 
 
+#: Output target for codegen - either embedded in HTML, or outputs to a typescript file. This is used to control
+#: the level of escaping that is done. When target is 'html', the code will be escaped for use in HTML.
+CodegenTarget = Literal["html"] | Literal["file"]
+
+#: When target is 'file', these characters will be escaped
+ESCAPE_TARGET_FILE = {
+    ord("\\"): "\\\\",
+    ord('"'): '\\"',
+    ord("'"): "\\'",
```

Comment:
may want to do "`" too?

Reply:
thanks - not needed for the general case, but codegen does support `TemplateExpression` which should be escaping strings including `. Fixed in 9ea5121